koku/api/report/ocp/query_handler.py

```diff
@@ -273,6 +273,9 @@ def execute_query(self):  # noqa: C901
 
             if self.is_csv_output:
                 data = list(query_data)
+                if self._report_type == "virtual_machines":
+                    # Handle formating OCP VM response
+                    data = self.format_vm_csv_response(query_data)
```

Comment:
It would be nice to skip the `list` conversion if we don't need to do that:
```suggestion
                if self._report_type == "virtual_machines":
                    # Handle formating OCP VM response
                    data = self.format_vm_csv_response(query_data)
                else:
                    data = list(query_data)
```